jwedb/tests/test_edb_interface.py

```diff
@@ -38,6 +38,26 @@
 from ..edb_interface import mnemonic_inventory, query_single_mnemonic, is_valid_mnemonic
 
 
+@pytest.mark.xfail(raises=(RuntimeError, FileNotFoundError, TypeError))
```

Comment:
Is this `mark.xfail` here because this test can't run on Jenkins? If so, I might change it so that it's a `mark.skipif` instead. This test as well as the 4th xfailed locally on my machine, and I'm not sure whether that is because it is supposed to fail or because something is broken.

Reply:
Thanks, that is a very good point. These tests will fail when they are run by Travis because it won't have the mast token stored in a local `.netrc` file. That was probably also the case for you.
I changed this to be skipped by Travis.